src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java

```diff
@@ -50,6 +50,9 @@ public OreCrusher(ItemGroup itemGroup, SlimefunItemStack item) {
 
     @Override
     protected void registerDefaultRecipes(List<ItemStack> recipes) {
+        recipes.add(new ItemStack(Material.BLACKSTONE, 8));
+        recipes.add(new ItemStack(Material.RED_SAND, 1));
```

Comment:
Are we sure that this is super logical? The cobblestone to sand makes sense to me but imo blackstone to red sand does not. Could just be me though

Reply:
While the relation ship from Blackstone > Sand isn't the best, it IS nice to have some variation in output, and add a way to renew a new type of... boring(?) item.

Reply:
I was doing it for equivalency, really.